mlc_llm/setup.py

```diff
@@ -0,0 +1,47 @@
+from distutils.core import setup
+from setuptools.dist import Distribution
```

Comment:
Given mlc_llm is the main package of the repo, maybe we do not need to move things to a subfolder, and just keep the current folder structure, which also means ppl can run without setting up any path